.build-test-rules.yml

```diff
@@ -1,4 +1,4 @@
 examples:
   disable:
-    - if: IDF_VERSION_MAJOR < 5 and CONFIG_NAME not in ["esp-box", "esp-box-lite", "esp32_s3_eye"]
+    - if: IDF_VERSION_MAJOR < 5 and CONFIG_NAME not in ["esp-box", "esp-box-lite", "esp32_s3_eye", "esp32_s3_korvo_2"]
```

Comment:
Just a comment for future: Once most BSPs support IDF v4.4.5, we should change this rule to list BSPs that do _not_ support IDF v4.4.5

Reply:
Sure, I saw it in your PR draft. But now it was easier to add done boards.